docs/en/week08/08-3.md

```diff
@@ -0,0 +1,198 @@
+
+Instead, we use the *reparameterization trick* to "sample" $\boldsymbol{z}$. We use $\boldsymbol{z} = \mathbb{U}(\boldsymbol{z}) + \boldsymbol{\epsilon} \odot \sqrt{\mathbb{V}(\boldsymbol{z})}$ where $\epsilon\sim \mathcal{N}(\boldsymbol{0}, \boldsymbol{I}_d)$. In this case, backpropagation in training is possible. Specifically, the gradients will go through the (element-wise) multiplication and addition in the above equation.
+
+**Visualizing Latent Variable Estimates and Reconstruction Loss**
+
+As stated above, the Free Energy for the VAE contains two parts: a reconstruction term and a regularization term.  We can write this as
+
+$$
+\tilde{F}(\boldsymbol{y}) = C(\boldsymbol{y}, \tilde{\boldsymbol{y}}) + \beta D_{\text{KL}}(\mathcal{N}(\boldsymbol{\mu}, \boldsymbol{v}) || \mathcal{N}(\boldsymbol{0}, \boldsymbol{1}))
+$$
+
+To visualize the purpose of each term in the free energy, we can think of each estimated $\boldsymbol{z}$ value as a circle in $2d$ space, where the centre of the circle is $\mathbb{U}(\boldsymbol{z})$ and the surrounding area are the possible values of $\boldsymbol{z}$ determined by $\mathbb{V}(\boldsymbol{z}).$
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/bubbles_z.png" height="350px" /><br>
+<b>Fig. 6</b>: Visualizing vector z as bubbles in the latent space
+</center>
+
+*In Fig. 6 above, each bubble represents an estimated region of $\boldsymbol{z}$, and the arrows represent how the reconstruction term pushes each estimated value away from the others, which is explained more below.*
+
+If there is an overlap between any two estimates of $z$, (visually, if two bubbles overlap) this creates ambiguity for reconstruction because the points in the overlap can be mapped to both original inputs. Therefore the reconstruction loss will push the points away from one another.
+
+However, if we use just the reconstruction loss, the estimates will continue to be pushed away from each other and the system could blow up.  This is where the penalty term comes in.
+
+**The penalty term**
+
+The second term is the relative entropy (a measure of the distance between two distributions) between $\boldsymbol{z}$ which comes from a Gaussian with mean $\mathbb{U}(\boldsymbol{z})$ referring to $\mu$, variance $\mathbb{V}(\boldsymbol{z})$ referring to $v$ and the standard normal distribution. If we expand this second term in the VAE loss function we get:
+
+$$
+\beta D_{\text{KL}}(\mathcal{N}(\boldsymbol{\mu}, \boldsymbol{v}) || \mathcal{N}(\boldsymbol{0}, \boldsymbol{1})) = \frac{\beta}{2} \sum\limits_{i=1}^d(v_i - \log{(v_i)} - 1 + \mu_i^2)
+$$
+
+Where each expression in the summation has four terms. Below we write out and graph the first three terms in *Fig. 7*.
+
+$$
+\mathbb{V} = v_i - \log{(v_i)} - 1
+$$
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/relative_entropy.png" /><br>
+<b>Fig. 7</b>: Plot showing how relative entropy forces the bubbles to have variance = 1
```

Comment:
This is wrong. Please, use NEW SLIDES. The content is **different** from last year.

Reply:
fixed